dp/cloud/docker/postgresql/create_magma_dbs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+declare -a DBS=(
```

Comment:
having an array with just one value looks like unnecessary complexity

Reply:
I create an array here, to easily extend it in the future, but You are right, it not necessary. I simplified it